src/routes.ts

```diff
@@ -238,18 +230,11 @@ export default [
     }
   },
   DigitalCredentialRoutes,
-  // {
-  //   path: '/about',
-  //   name: 'about',
-  //   // route level code-splitting
-  //   // this generates a separate chunk (about.[hash].js) for this route
-  //   // which is lazy-loaded when the route is visited.
-  //   // ref: https://cli.vuejs.org/guide/html-and-static-assets.html#prefetch
-  //   component: () => import(/* webpackChunkName: "about" */ './views/About.vue')
-  // },
```

Comment:
I'm OK with deleting this. It was an idea way back when for lazy loading components using Webpack. This is now obsolete.